vlm/src/main/scala/geotrellis/contrib/vlm/RasterSummary.scala

```diff
@@ -15,13 +31,13 @@ case class RasterSummary(
   extent: Extent,
   cells: Long,
   count: Long
-) {
+) extends LazyLogging {
```

Comment:
An aside: how do you feel about this pattern? 
I'm not sure if its "bad" but it feels weird to include this in type definition just to get a private field for `logger`.

Reply:
Well, i dont feel really bad because of it; as its what is suggested by scalalogging: https://github.com/lightbend/scala-logging/blob/d271c40599bd6cc4dcee35b90768506f50b26e33/src/main/scala/com/typesafe/scalalogging/Logging.scala#L25 

Also it looks like from 3.9.0 it is transient, so we can remove this trait in 3.0!